tests/Search/SearchResultTest.php

```diff
@@ -229,4 +229,25 @@ public function testTransformFacetsDistributionMethod(): void
         $this->assertEquals(1, $response->getFacetDistribution()['genre']['FANTASY']);
         $this->assertEquals(1, $response->getFacetDistribution()['genre']['ADVENTURE']);
     }
+
+    public function testRonaldo(): void
```

Comment:
Why this name of test?

Reply:
It was a mistake it shouldn't be commited, nevermind 🤦